pkg/registry/networking/servicecidr/strategy_test.go

```diff
@@ -15,3 +15,117 @@ limitations under the License.
 */
 
 package servicecidr
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/kubernetes/pkg/apis/networking"
+	"sigs.k8s.io/structured-merge-diff/v4/fieldpath"
+)
+
+func TestNamespaceScoped(t *testing.T) {
+	strategy := serviceCIDRStrategy{}
+	if strategy.NamespaceScoped() {
+		t.Errorf("Expected ServiceCIDRStrategy to be cluster-scoped")
+	}
+}
+
+func TestGetResetFields(t *testing.T) {
+	strategy := serviceCIDRStrategy{}
+	fields := strategy.GetResetFields()
+	status := fieldpath.NewSet(
+		fieldpath.MakePathOrDie("status"),
+	)
+	if !fields["networking/v1alpha1"].Equals(status) {
+		t.Errorf("Expected 'status' field to be reset for networking/v1alpha1")
+	}
+}
```

Comment:
I prefer the existing mode in other places that does not depend on the api group and version

```go
func TestStrategy_ResetFields(t *testing.T) {
	resetFields := Strategy.GetResetFields()
	if len(resetFields) != 1 {
		t.Errorf("ResetFields should have 1 element, but have %d", len(resetFields))
	}
}
```

Reply:
Yes , it's better.